### PR TITLE
gmcp efun/apply documentation

### PR DESCRIPTION
**Pull request by GwennKoi**
There wasn't any so I took a shot at it. Please let me know if I need to change anything.

### CODE REVIEW

File: docs/efun/interactive/send_gmcp.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: interactive / send_gmcp
+---
+
+### NAME
+
+    send_gmcp() - sends a message to all living objects
```
Comment (thefallentree) [inline docs/efun/interactive/send_gmcp.md:8]: This sentence seems wrong.

Comment (GwennKoi) [reply docs/efun/interactive/send_gmcp.md:8]: oops, used shout() as my template. I'll fix.

File: docs/efun/interactive/has_gmcp.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: interactive / has_gmcp
+---
+
+### NAME
+
+    has_gmcp() - detects whether or not a given interactive user's client
```
Comment (thefallentree) [inline docs/efun/interactive/has_gmcp.md:8]: I would not say it “detects” , just “returns”

this function doesn’t trigger gmcp negotiation, it just tells if this client has gmcp enabled

Comment (GwennKoi) [reply docs/efun/interactive/has_gmcp.md]: fixed in 2690f8b

File: docs/efun/interactive/send_gmcp.md (hunk 1 above)
Comment (GwennKoi) [reply docs/efun/interactive/send_gmcp.md]: Fixed in 3a47063

**Review by thefallentree — APPROVED** [review]

**Pull request merged**